docs/source/releasing.rst

```diff
@@ -21,3 +21,24 @@ Making a new release
 #. Make a release on Github based on the pushed tag.
    Copy the relevant ``CHANGELOG.md`` part to the release notes;
    see `this page <https://github.com/beamer-bridge/beamer/releases/tag/v0.1.8>`_ for example.
+
+Making a new deployment npm package release
+-------------------------------------------
+
+The *@beamer-bridge/deployments* package is a npm package that contains the deployment ABIs and addresses.
+It also contains a *git-commit-version.txt* file that contains the commit hash of the commit that was used to create the package.
+
+There are 2 different release pipelines for this package.
+
+---------
+Automatic
+---------
+Whenever a PR changes something inside the deployments folder, a new dev release is created on circleCI and pushed to npm.
+
+------
+Manual
+------
+Whenever we want or need to pin a new major version of the deployments ABIs and addresses, we need to manually update the *package.json* version and create a PR. Once the PR is merged into main, a new release with the specified version number is created on circleCI and pushed to npm.
```

Comment:
Could you please break this very long line so that it looks more reasonable?

Reply:
Also `circleCI` should be `CircleCI`.

Reply:
sorry, about that. I was missing the ruler in VSCode and since wordwrap was on I was not realizing how long the line was.
I had to configure vscode to display a ruler at 80 characters:
<img width="276" alt="image" src="https://github.com/beamer-bridge/beamer/assets/693770/41144f4f-c25d-497a-a1e4-ef17248e503b">